codegen/projections/white_label/spec/event_stream_stubs_spec.rb

```diff
@@ -0,0 +1,305 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+describe WhiteLabel do
+  let(:event_message) { 'event_message' }
+  let(:event_header) { 'event_header' }
+  let(:initial_message) { 'initial_message' }
+  let(:complex_data) { { values: %w[a b c] } }
+  let(:event_handler) { WhiteLabel::EventStream::StartEventStreamHandler.new }
+  let(:handler) { proc {} }
+
+  subject do
+    WhiteLabel::Client.new(stub_responses: true)
+  end
+
+  context 'event stub Message' do
+    let(:message) do
+      headers = {
+        ':message-type' => 'event',
+        ':event-type' => 'SimpleEvent'
+      }
+      headers.each do |k, v|
+        headers[k] =
+          Hearth::EventStream::HeaderValue.new(value: v, type: 'string')
+      end
+      Hearth::EventStream::Message.new(
+        headers: headers,
+        payload: StringIO.new('{"message":"event_message"}')
+      )
+    end
+
+    it 'signals the stubbed event' do
+      subject.stub_responses(:start_event_stream, {
```

Comment:
Wonky formatting on all of these.

Reply:
ugh, rubocop kept mucking these up.  I think I've fixed them all.